pages/index.tsx

```diff
@@ -228,55 +56,10 @@ const IndexPage = ({ data }: { data: TLandingPage }) => {
         />
         <meta name="twitter:card" content="summary_large_image"></meta>
       </Head>
-      <ModalWrapper ref={modalWrapperRef}>
-        <Modal>
-          <MessageWrapper>
-            <ImgWrapper src="/icons/contact-email.svg" alt="" />
-            <ModalHeading>Contact Us Via Email</ModalHeading>
-            <p>
-              Any questions? Send us an email at prolog@profy.dev. We usually
-              answer within 24 hours.
-            </p>
-          </MessageWrapper>
-          <ButtonWrapper>
-            <CustomButton
-              size={ButtonSize.sm}
-              color={ButtonColor.gray}
-              onClick={cancelHandler}
-            >
-              Cancel
-            </CustomButton>
-            <CustomButton
-              size={ButtonSize.sm}
-              color={ButtonColor.primary}
-              onClick={emailHandler}
-            >
-              Open Email App
-            </CustomButton>
-          </ButtonWrapper>
-        </Modal>
-      </ModalWrapper>
-      <Header>
-        {/* eslint-disable-next-line @next/next/no-img-element */}
-        <img src="/icons/logo-large.svg" alt="Prolog logo" />
-        <MenuWrapper isMobileMenuOpen={isMobileMenuOpen}>
-          <Navigation>
-            <Link href="/">Home</Link>
-            <Link href="/products">Products</Link>
-            <Link href="/documentation">Documentation</Link>
-            <Link href="/pricing">Pricing</Link>
-          </Navigation>
-          <OpenDashboardLink href={Routes.projects}>
-            Open Dashboard
-          </OpenDashboardLink>
-        </MenuWrapper>
-        <MenuIconWrapper>
-          <CustomButton size={ButtonSize.sm} onClick={toggleMobileMenu}>
-            {/* eslint-disable-next-line @next/next/no-img-element */}
-            <img src="/icons/hamburger.svg" alt="open navigation menu" />
-          </CustomButton>
-        </MenuIconWrapper>
-      </Header>
+
+      {isModalVisible && <Modal setIsModalVisible={setIsModalVisible} />}
```

Comment:
It might be a good idea to use a context for the modal here. Then you wouldn't have to add the `isModalVisible` to this component and pass down the `setIsModalVisible` handler.